src/PreallocationTools.jl

```diff
@@ -204,18 +204,17 @@ same type and size `f(size(u))` (defaulting to the same size), which is allocate
 needed and then cached within `b` for subsequent usage.
 """
 struct LazyBufferCache{F <: Function}
-    bufs::Dict # a dictionary mapping types to buffers
+    bufs::Dict{Any, Any} # a dictionary mapping (type, size) pairs to buffers
     sizemap::F
     LazyBufferCache(f::F = identity) where {F <: Function} = new{F}(Dict(), f) # start with empty dict
 end
 
 # override the [] method
 function Base.getindex(b::LazyBufferCache, u::T) where {T <: AbstractArray}
     s = b.sizemap(size(u)) # required buffer size
-    buf = get!(b.bufs, (T, s)) do
+    get!(b.bufs, (T, s)) do
```

Comment:
Does that change actually make a difference?

Reply:
Not at all. That is just for stylistic consistency with the rest of the file.